ModelBase/src/model/TreeManager.h

```diff
@@ -490,6 +506,9 @@ inline Node* TreeManager::root(){ return root_; }
 inline QString TreeManager::name() { return name_; }
 inline void TreeManager::setName(const QString& name) { name_ = name; }
 
+inline QString TreeManager::managerName() { return managerName_; }
```

Comment:
I could also call it `revisionName` and only distinguish between new and old. That would also work.
BTW: is `TreeManager::name` not also a bad name, since it is not the name of the manager but the name of the tree that is managed? (At least if I understood correctly)